utils/db/teams/addActionCount.ts

```diff
@@ -1,8 +1,8 @@
 import executeRequest from "../azuredb";
 
-export default async ({ watermelon_user }) => {
+export default async ({ owner }) => {
   let query = await executeRequest(
-    `EXEC dbo.increment_github_app_uses @watermelon_user = '${watermelon_user}'`
+    `EXEC dbo.increment_owner_github_app_uses @watermelon_user = '${owner}'`
```

Comment:
How does this work for forks?

Reply:
For example, in this repo, people need to fork it to open a PR, but when the PR is run it will count against the "watermelontools" owner, not the contributor one, and we'll miss the chance of adding a new user.

Reply:
First of all, we don't limit open-source usage so if people are forking a repo of such kind **it doesn't matter.** 

But even if we did, or in the future we decide that we want to do so, we can choose not to add to the count if the repo is a fork. Octokit allows us to determine if a repo is a fork or not via various methods. One of them could be calling `octokit.rest.repos.get` and checking if the `parent` and `source` props exist for that repo

Reply:
Stamp.